tests/api/admin/controller/test_report.py

```diff
@@ -36,15 +39,3 @@ def test_generate_inventory_report(self, report_fixture: ReportControllerFixture
             assert response.status_code == HTTPStatus.ACCEPTED
             body = json.loads(response.data)  # type: ignore
             assert body and body["message"].__contains__("admin@email.com")
```

Comment:
Minor: Probably better to use the `in` operator here, rather then directly calling `__contains__`.